CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+0.13.1 / 2017-10-11
```

Comment:
The removal of the cache feel more like a feature than a bug fix to me. If I was adding it I would certainly be thinking it would be a minor version increase as opposed to a patch version.

Reply:
Agreed. Have upped the minor version instead